src/pytorch_lightning/plugins/precision/deepspeed.py

```diff
@@ -97,24 +86,19 @@ def backward(
                 "You have overridden the `LightningModule.backward` hook but it will be ignored since DeepSpeed handles"
                 " the backward logic internally."
             )
+        optimizer, optimizer_idx, *args = args
```

Comment:
These are unused

Reply:
I prefer this way to make it explicit what the two arguments we are unpacking do.